docs/source/concepts.rst

```diff
@@ -260,6 +260,25 @@ These events could be used to attach any handler and are fired using :meth:`~ign
    See the source code of :class:`~ignite.contrib.engines.create_supervised_tbptt_trainer` for an example of usage of
    custom events.
 
+If you want to use filtering with custom events (e.g. with `CallableEventWithFilter`), you need to do 3 more things:
```

Comment:
Can you also please add a subsection to the line 229 : 

    Custom events
    ``````````````

Reply:
Done!